jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -232,9 +244,30 @@ def vue_do_aper_phot(self, *args, **kwargs):
                     d['id'] = 1
                     self.app._aper_phot_results = _qtable_from_dict(d)
 
+            # Radial profile
+            if plt is None:
+                self.radial_plot = 'radial plot N/A: install matplotlib'

```

Comment:
I don't think this will work with the img tag, but you could add a trait like 'error = ' and do an `v-if="error"` in the template.

Reply:
Good point... 😆